src/Widgets/WifiInterface.vala

```diff
@@ -22,6 +22,8 @@ public class Network.WifiInterface : Network.WidgetNMInterface {
     public NM.DeviceWifi? wifi_device;
     public bool hidden_sensitivity { get; set; default = true; }
 
+    public string active_ap_name { get; private set; }
```

Comment:
Same thing here. `display_title` should already be set for this class, and if it isn't that property exists for this purpose

Reply:
For wireless, I don't think we should set/use `display_title` since this is used in the switch label 

https://github.com/elementary/wingpanel-indicator-network/blob/19e38f656827eac52648a97265f9a3012f96ba96/src/Widgets/WifiInterface.vala#L70

and setting it will change the switch label to the name of the connected network, unless this is intended, similar to how "Wired" changes to the connected ethernet device.

Reply:
Oh you know what, you're right. Because for Wireless we want the AP name and not the name of the device. Sorry, ignore me. Holiday brain